### DISCUSSION TITLE
Deploy adonis on shared serveur

### DISCUSSION THREAD

**Discussion by MaoFofana** [adonisjs/core#2158] [General]
Hello, I am trying to deploy adonis on a shared server but I am experiencing an error. I don't know if it's the version of node that is version 10 or it's because I made a bad configuration.
Here are the deployment steps
1 - I do node ace build --production
 to compile the project
2 - I copy the contents of the build folder into the folder of the node application that I created on the server
3- I create the environment variable
4 - I install the dependencies
 but I get an error {"message": "this.scopes.flat is not a function"}
Thank you in advance for your help

**Comment by steschwa** [comment]
Not sure if the Node version causes this error but according to the Adonis Docs `Node >= 12.0.0` is required. https://preview.adonisjs.com/guides/quick-start#creating-a-new-project
